content/node-placement/docs/fargate-node-placement.md

```diff
@@ -0,0 +1,178 @@
+
+A Spark application whose driver and executor pods are distributed across multiple AZs can incur inter-AZ data transfer costs. To minimize or eliminate inter-AZ data transfer costs, you can configure the application to only run on the nodes within a single AZ.  In this example, we use the kubernetes [node selector](https://spark.apache.org/docs/latest/running-on-kubernetes.html#pod-spec) to specify which AZ should the job run on.
+
+
+**Create Fargate Profile**
+Create your Fargate profile with the following eksctl command, replacing the `<variable text>` (including <>) with your own values. You're required to specify a namespace. However, the `--labels` option is not required.
+
+```
+eksctl create fargateprofile \
+    --cluster <cluster_name> \
+    --name <fargate_profile_name> \
+    --namespace <virtual_cluster_mapped_namespace> \
+    --labels spark-node-placement=fargate
+```
+
+### 1- Place entire job including driver pod on Fargate 
+
+When both Driver and Executor are labeled that matches the Fargate Selector, the entire job including the driver pod could run on Fargate.
+
+**Request:**
+```
+cat >spark-python-in-s3-nodeselector.json << EOF
+{
+  "name": "spark-python-in-s3-fargate-nodeselector", 
+  "virtualClusterId": "<virtual-cluster-id>", 
+  "executionRoleArn": "<execution-role-arn>", 
+  "releaseLabel": "emr-6.3.0-latest", 
+  "jobDriver": {
+    "sparkSubmitJobDriver": {
+      "entryPoint": "s3://<s3 prefix>/trip-count.py", 
+       "sparkSubmitParameters": "--conf spark.driver.cores=4  --conf spark.executor.memory=20G --conf spark.driver.memory=20G --conf spark.executor.cores=4"
+    }
+  }, 
+  "configurationOverrides": {
+    "applicationConfiguration": [
+      {
+        "classification": "spark-defaults", 
+        "properties": {
+            "spark.kubernetes.driver.label.spark-node-placement": "fargate",
+            "spark.kubernetes.executor.label.spark-node-placement": "fargate"
+         }
+      }
+    ], 
+    "monitoringConfiguration": {
+      "cloudWatchMonitoringConfiguration": {
+        "logGroupName": "/emr-containers/jobs", 
+        "logStreamNamePrefix": "demo"
+      }, 
+      "s3MonitoringConfiguration": {
+        "logUri": "s3://joblogs"
+      }
+    }
+  }
+}
+EOF
+aws emr-containers start-job-run --cli-input-json file:///spark-python-in-s3-nodeselector.json
+```
+
+**Observed Behavior:**  
+When the job starts the driver pod and executor pods are scheduled only on fargate since both are labeled with the `spark-node-placement: fargate`. This is useful when we want to run the entire job on Fargate nodes. The maximum vCPU available for the driver pod is 4vCPU. 
```

Comment:
```suggestion
When the job starts, the driver pod and executor pods are scheduled only on Fargate since both are labeled with the `spark-node-placement: fargate`. This is useful when we want to run the entire job on Fargate nodes. The maximum vCPU available for the driver pod is 4vCPU. 
```

Reply:
Addressed.